info.plist

```diff
@@ -105,11 +98,36 @@ require "net/http"
 require "uri"
 require "json"
 
-uri = URI.parse("https://pwpush.com/p.json")
+uri = URI.parse(ENV['site'] + "/p.json")
```

Comment:
It's a bit verbose, but let's use the [same environment variables](https://github.com/pglombardo/PasswordPusher/blob/5e87449439c9f282a45423a55c5b3f41bdd15f36/Configuration.md#proxies) as Password Pusher: `PWP__OVERRIDE_BASE_URL`